app/bundles/CampaignBundle/Entity/Event.php

```diff
@@ -833,4 +847,26 @@ public function addContactLog($contactLog)
 
         return $this;
     }
+    
+    /** CAPTIVEA.CORE START **/
+    
+    /**
+     * 
+     * @return \Mautic\ScoringBundle\Entity\ScoringCategory
+     */
+    public function getScoringCategory() {
+       return $this->scoringCategory; 
+}
+    
+    /**
+     * 
+     * @param \Mautic\ScoringBundle\Entity\ScoringCategory $scoringCategory
+     * @return $this
+     */
+    public function setScoringCategory(\Mautic\ScoringBundle\Entity\ScoringCategory $scoringCategory = null) {
```

Comment:
Please define the namespaces used in the class at the top of the class with the `use` statement. It will save some space and other devs will know what classes are used at one glance.